common/app/views/support/cleaner/VideoEmbedCleaner.scala

```diff
@@ -8,7 +8,10 @@ import views.support.{HtmlCleaner, Item640}
 
 import scala.collection.JavaConversions._
 
-case class VideoEmbedCleaner(article: Article) extends HtmlCleaner {
+/*
+ * maxEmbedHeight: 812px - full height on an iPhone X
+ */
```

Comment:
Does that mean we will have to change this value next time Apple release an new phone? 📱

Reply:
It was suggested as a reasonable default for the time being but I guess it depends how big they go!